src/popup/components/AccountDetailsNavigation.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+  <div>
```

Comment:
The rule that Nikita mentioned many times is to add class name equal to the component's file name to the root element.